### PR TITLE
Migrate cgroup-v2 kep to new template

### PR DESCRIPTION
**Pull request by giuseppe**
ref #2220

Signed-off-by: Giuseppe Scrivano <gscrivan@redhat.com>

### CODE REVIEW

File: keps/sig-node/1370-cgroup-v2/kep.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+title: cgroups v2
```
Comment (wojtek-t) [inline keps/sig-node/1370-cgroup-v2/kep.yaml:1]: It shouldn't be KEP number 1370.

You should open an issue in this repo, and the KEP number should be the number of that issue.

Comment (giuseppe) [reply keps/sig-node/1370-cgroup-v2/kep.yaml]: thanks, opened a new issue and linked here

**Comment by wojtek-t** [comment]
Thank you!

/lgtm
/approve

**Comment by derekwaynecarr** [comment]
thanks @giuseppe 

/approve
/lgtm

**Pull request merged**